docs/embedding/sdk/introduction.md

```diff
@@ -20,10 +20,10 @@ To give you and idea of what's possible with the SDK, we've put together example
 
 ## Embedded analytics SDK prerequisites
 
-- React application. The SDK is tested to work with React 18 or higher, though it may work with earlier versions.
+- React application. The SDK is tested to work with React 18 and Next.js 14, though it may work with earlier versions.
 - [Metabase Pro or Enterprise subscription or free trial](https://www.metabase.com/pricing/).
 - Metabase version 1.51 or higher.
-- [Node.js 18.x LTS](https://nodejs.org/en) or higher.
+- [Node.js 18.x LTS](https://nodejs.org/en).
```

Comment:
Current LTS is [Node 22](https://nodejs.org/en/blog/announcements/v22-release-announce), and Node 18 will be end-of-life in 3 months. I am currently using Node 20.15.0 without any issues, so I would recommend bumping LTS here to 20.x LTS

Reply:
OK